core/miner/storage_fsm/impl/sealing_impl.cpp

```diff
@@ -745,9 +751,6 @@ namespace fc::mining {
                 [](auto info, auto event, auto context, auto from, auto to) {
                   info->invalid_proofs++;
                 }),
-        SealingTransition(SealingEvent::kSectorRetryCommitWait)
```

Comment:
No such rule in Lotus.
Only 
[CommitFailed -(SectorRetryCommitWait)-> CommitWait](https://github.com/filecoin-project/lotus/blob/master/extern/storage-sealing/fsm.go#L233)
and
[Committing -(SectorRetryCommitWait)-> CommitWait](https://github.com/filecoin-project/lotus/blob/master/extern/storage-sealing/fsm.go#L664)

Reply:
Both defined [here](https://github.com/filecoin-project/cpp-filecoin/blob/master/core/miner/storage_fsm/impl/sealing_impl.cpp#L695).
kComputeProof -(SectorRetryCommitWait)-> CommitWait
is wrong?

Reply:
Yes, that's bug